packages/containers/src/Notification/pushNotification.js

```diff
@@ -7,9 +7,19 @@ export default function pushNotification(state, notification) {
 	}
 	const path = ['Container(Notification)', 'Notification', 'notifications'];
 	let notifs = state.cmf.components.getIn(path);
-	notifs = notifs.push(Object.assign({
-		id: objectId(),
-	}, notification));
+	if (!notifs) {
+		// eslint-disable-next-line no-console
+		console.error('Notifications are not yet available.', notification);
```

Comment:
Don't we prefer the use of invariant instead of console ?

Reply:
I am not fan of invariant invariant(true, ...) and invariant(notifs, ...) will only evaluate a second time the content of the if statement

Reply:
I agree with @nicomaligne here. Please use `invariant` instead of plain `console.log`. It has many advantages over it.